mlflow/pyfunc/__init__.py

```diff
@@ -486,6 +486,67 @@ def load_model(
     return PyFuncModel(model_meta=model_meta, model_impl=model_impl, predict_fn=predict_fn)
 
 
+class ModelServerPyfunc:
+    def __init__(self, client):
+        self._client = client
+
+    def predict(self, _input):
+        result = self._client.invoke(_input)
+        if isinstance(result, pandas.DataFrame):
+            result = result[result.columns[0]]
```

Comment:
Is this always the correct thing to do? For example, Spark models write predictions to a DataFrame column called `prediction`.